File: springboot-app/src/main/java/com/lumos/lumosspring/dto/stock/StockMovementDTO.java
package com.lumos.lumosspring.dto.stock;

import java.math.BigDecimal;

public record StockMovementDTO(String description,
                               Long materialId,
                               BigDecimal inputQuantity,
                               String buyUnit,
                               String requestUnit,
                               BigDecimal quantityPackage,
                               String priceTotal,
                               String supplierId,
                               BigDecimal totalQuantity) {
}

File: angular-app/src/app/stock/dto/stock-movement.dto.ts
export interface StockMovementDTO {
    materialStockId: number;
    materialName: string;
    barcode: string | null;
    description: string;
    inputQuantity: string;
    buyUnit: string;
    requestUnit: string;
    quantityPackage: string;
    priceTotal: string;
    totalQuantity: string;
    hidden: boolean;
    invalid: boolean;
}
